Description: Omxdec: Workaround for user that does not release all output buffers
 .
 ffmpeg (7:6.1.1) UNRELEASED; urgency=medium
 .
   * libavcodec/omxdec: upgrade sf-omx-vpu from 5.1.x to 6.1.x.
Author: Leo Lu <leo.lu@starfivetech.com>


--- a/libavcodec/omxdec.c
+++ b/libavcodec/omxdec.c
@@ -611,7 +611,7 @@
         av_log(logctx, AV_LOG_WARNING, "No component for role %s found\n", role);
         return AVERROR_DECODER_NOT_FOUND;
     }
-    components = av_mallocz_array(num, sizeof(*components));
+    components = av_calloc(num, sizeof(*components));
     if (!components)
         return AVERROR(ENOMEM);
     for (i = 0; i < num; i++) {
